app/controllers/changes_controller.rb

```diff
@@ -23,6 +23,34 @@ def resend
     render json: @change.send_notifications
   end
 
+  def html
+    @change_id=params[:change_id]
+    @snapshot = PageSnapshot.find(@change_id)
+    @page = Page.find(@snapshot.page_id)
+    @timestamp = @snapshot.created_at
+    @html = @snapshot.html
```

Comment:
Similarly I wouldn't assign these pieces of data to their own variables, it just obfuscates where the data is coming from to someone looking at the view.